imports/ui/case-explorer/case-explorer.jsx

```diff
@@ -3,7 +3,7 @@ import { Meteor } from 'meteor/meteor'
 import { connect } from 'react-redux'
 import { createContainer } from 'meteor/react-meteor-data'
 import PropTypes from 'prop-types'
-import UnverifiedWarning from '../components/unverified-warning.jsx'
+import UnverifiedWarning from '../unverified-warning/unverified-warning'
```

Comment:
Why was *unverified-warning.jsx* moved to its own folder? separate folders are usually reserved (by our loose conventions) for top level route views. I think it should stay in the "components" folder